src/molecules/formfields/DateField/index.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import dateUtils from 'utils/dateUtils';
+import renderOption from 'utils/Fields/renderOption';
+
+import styles from 'molecules/formfields/shared/formfields.module.scss';
+
+const renderMonthOption = (month, idx) => renderOption({
```

Comment:
@patrickkim It would be nice great to have better test coverage on all of these functions.